src/TWI.hpp

```diff
@@ -24,7 +24,7 @@ class TWI {
      *
      * Function calculates the TWI timings required to reach the specified speed.
      */
-    constexpr auto setClock() -> void {
+    constexpr auto setClock() noexcept -> void {
         uint32_t ckdiv  = 0; ///< Clock divider
```

Comment:
Inconsistency in naming. ckdiv on one hand. cLHDiv on the other. Recommend ckDiv.